src/config/index.ts

```diff
@@ -1,30 +1,32 @@
 import dotenv from "dotenv";
 
-// Set the NODE_ENV to 'development' by default
+interface Config {
+  port: number;
+  api: {
+    prefix: string;
+  };
+  logs: {
+    level: string;
+  };
+}
+
 process.env.NODE_ENV = process.env.NODE_ENV || "development";
 
-const envFound = dotenv.config();
+const envFound = dotenv.config({
```

Comment:
@Extheoisah seems you forgot to change this

Reply:
Oh! yeah. I changed it in my follow up pr. If you want, we can merge this and apply the fix in my followup pr or I can make the change now @tobi-bams

Reply:
Sure lemme merge this!!!